### PR TITLE
Change model and mako for update wrz

### PR DESCRIPTION
**Pull request by Sgachet**
Update for wrz_portal and wrz_select : 
. added searchable attributes

Test link : https://mf-geoadmin3.dev.bgdi.ch/ltfap/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fltgas_wrz_portal_update&lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bafu.wrz-wildruhezonen_portal,ch.bafu.wrz-jagdbanngebiete_select

### CODE REVIEW

**Comment by pfanguin** [comment]
jenkins test

**Comment by pfanguin** [comment]
Sphinx config : https://github.com/geoadmin/service-sphinxsearch/pull/294

**Comment by loicgasser** [comment]
WIP not WIP? Please adapt the title when the PR is ready. Thanks!

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -1946,15 +1946,21 @@ class wrzselect(Base, Vector):
     __table_args__ = ({'schema': 'wrzportal', 'autoload': False})
     __bodId__ = 'ch.bafu.wrz-jagdbanngebiete_select'
     __template__ = 'templates/htmlpopup/wrz_select.mako'
+    __queryable_attributes__ = ['kanton', 'jb_name', 'beschlussjahr', 'schutzs_de', 'schutzs_fr', 'schutzs_it']
     __label__ = 'jb_name'
     id = Column('objectid', Integer, primary_key=True)
     jb_name = Column('jb_name', Text)
-    jb_obj = Column('jb_obj', Integer)
-    schutzstatus = Column('schutzstatus', Text)
+    schutzs_de = Column('schutzs_de', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/bafu.py]: Unicode instead? Everywhere you use text.

File: chsdi/models/vector/bafu.py [hunk 2]
```diff
@@ -1946,15 +1946,21 @@ class wrzselect(Base, Vector):
...
+    schutzs_it = Column('schutzs_it', Text)
+    schutzs_fr = Column('schutzs_fr', Text)
     bestimmung = Column('bestimmung', Text)
+    best_de = Column('best_de', Text)
+    best_it = Column('best_it', Text)
+    best_fr = Column('best_fr', Text)
     schutzzeit = Column('schutzzeit', Text)
     grundlage = Column('grundlage', Text)
     zusatzinformation = Column('zusatzinformation', Text)
+    beschlussjahr = Column('beschlussjahr', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/bafu.py]: I would use integers for years.

Comment (loicgasser) [inline chsdi/models/vector/bafu.py]: Or even better a date format.

**Comment by pfanguin** [comment]
It's done

**Pull request merged**